src/exec/run-batch-submitter.ts

```diff
@@ -164,18 +173,29 @@ export const run = async () => {
         if (pendingTxs > latestTxs) {
           log.info('Detected pending transactions. Clearing all transactions!')
           for (let i = latestTxs; i < pendingTxs; i++) {
-            const response = await sequencerSigner.sendTransaction({
-              to: await sequencerSigner.getAddress(),
-              value: 0,
-              nonce: i,
-            })
-            log.info(`Submitting transaction with nonce: ${i}; hash: ${response.hash}`)
```

Comment:
note: hash is no longer getting logged in this line since the transaction isn't submitted here